nabd/nabd.py

```diff
@@ -38,8 +38,8 @@ class State(Enum):
 
 
 class Nabd:
-    SLEEP_EAR_POSITION = 10
-    INIT_EAR_POSITION = 0
+    SLEEP_EAR_POSITION = 13
+    INIT_EAR_POSITION = 3
```

Comment:
Is there a definite reference on what were the values in the original Violet code ? (I can't find any)

Reply:
If I remember properly my research about it while I tried to set the positions accurately, the original code considered 2 as the init position instead of 3, but it heavily relied on the hardware timing and more importantly counted differently. Eventually, I believe this was a bug in original code which could be exhibited by crafting a specific choreography which would advance ears at positions 14, 15 and 14.

I tried to document it here:
https://github.com/pguyot/tagtagtag-ears/blob/master/tagtagtag-ears.c